code/datums/gamemode/role/syndicate.dm

```diff
@@ -132,60 +132,61 @@
 
-/datum/role/traitor/rogue/ForgeObjectives()
-	var/datum/role/traitor/rogue/rival
-	var/list/potential_rivals = list()
-	if(faction && faction.members)
-		potential_rivals = faction.members-src
-	else
-		for(var/datum/role/traitor/rogue/R in ticker.mode.orphaned_roles) //It'd be awkward if you ended up with your rival being a vampire.
-			if(R != src)
-				potential_rivals.Add(R)
-	if(potential_rivals.len)
-		rival = pick(potential_rivals)
-	if(!rival) //Fuck it, you're now a regular traitor
-		return ..()
-
-	var/datum/objective/target/assassinate/kill_rival = new(auto_target = FALSE)
-	if(kill_rival.set_target(rival.antag))
-		AppendObjective(kill_rival)
-	else
-		qdel(kill_rival)
+/datum/role/traitor/challenger/Greet(var/greeting,var/custom)
+	if(!greeting)
+		return
 
-	if(prob(70)) //Your target knows!
-		var/datum/objective/target/assassinate/kill_new_rival = new(auto_target = FALSE)
-		if(kill_new_rival.set_target(antag))
-			rival.AppendObjective(kill_new_rival)
+	var/icon/logo = icon('icons/logos.dmi', logo_state)
+	switch(greeting)
+		if (GREET_ROUNDSTART)
+			to_chat(antag.current, "<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <span class='danger'>You are a Syndicate Challenger. You have been pitched along with other volunteers into a battle royale aboard of one of Nanotrasen's space stations for the priviledge of becoming a fully fledged Syndicate agent. Take the other agents out before they do the same to you.</span>")
 		else
-			qdel(kill_new_rival)
+			to_chat(antag.current, "<img src='data:image/png;base64,[icon2base64(logo)]' style='position: relative; top: 10;'/> <span class='danger'>You are a Syndicate Challenger.</span>")
 
-	if(prob(50)) //Spy v Spy
-		var/datum/objective/target/assassinate/A = new()
-		if(A.target)
-			AppendObjective(A)
-
-			var/datum/objective/target/protect/P = new(auto_target = FALSE)
-			if(P.set_target(A.target))
-				rival.AppendObjective(P)
+	to_chat(antag.current, "<span class='info'><a HREF='?src=\ref[antag.current];getwiki=[wikiroute]'>(Wiki Guide)</a></span>")
 
-	if(prob(30))
-		AppendObjective(/datum/objective/target/steal)
 
-	switch(rand(1,3))
-		if(1)
-			if(!locate(/datum/objective/target/steal) in objectives.GetObjectives())
-				AppendObjective(/datum/objective/die)
-			else
-				AppendObjective(/datum/objective/escape)
-		if(2)
-			AppendObjective(/datum/objective/hijack)
+/datum/role/traitor/challenger/OnPostSetup()
+	. = ..()
+	maybe_equip(new /obj/item/device/camera(get_turf(antag.current)))
+
+/datum/role/traitor/challenger/proc/maybe_equip(obj/item/thing)
+	var/mob/living/carbon/human/mob = antag.current
+	if(ishuman(mob))
+		var/list/slots = list(
+			"backpack" = slot_in_backpack,
+			"left pocket" = slot_l_store,
+			"right pocket" = slot_r_store,
+		)
+		var/where = mob.equip_in_one_of_slots(thing, slots, put_in_hand_if_fail = 1)
+
+		if (!where)
+			to_chat(mob, "\The Syndicate was unfortunately unable to get you \a [thing].")
```

Comment:
You shouldn't use `\The` like that. It only makes sense in the context of string interpolation with objects, which you're not doing here. Just write out `"The Syndicate...` directly.

Reply:
Oh yeah that's a bad copypaste from rev code, I'll fix it there too.